### ISSUE TITLE
Clean builds with test fail with missing native components

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/coreclr#2273]
If you do a clean build from toplevel, it invariably fails building tests with errors like:
```
 C:\Repos\coreclr> build.cmd x64 release clean
 ...
  2665>C:\Repos\coreclr\tests\src\dir.targets(112,6): error : The native project files are missing in C:\Repos\coreclr\tests\..\bin\tests\obj\Windows_NT.x64.Release\src\Interop\ReversePInvoke\Marshalling\Release\ please run build from the root of the repo at least once [C:\Repos\coreclr\tests\src\Interop\ReversePInvoke\Marshalling\MarshalBoolArray.csproj]
  2665>Done Building Project "C:\Repos\coreclr\tests\src\Interop\ReversePInvoke\Marshalling\MarshalBoolArray.csproj" (CopyNativeProjectBinaries target(s)) -- FAILED.
  2665>Done Building Project "C:\Repos\coreclr\tests\src\Interop\ReversePInvoke\Marshalling\MarshalBoolArray.csproj" (Build target(s)) -- FAILED.
```
A rebuild then succeeds. 

Seems like we're missing a dependence somewhere or something.

**Comment by joshfree** [comment]
@yizhang82 PTAL

**Comment by yizhang82** [comment]
@tijoytom Can you take a look?

**Comment by AndyAyersMS** [comment]
Believe this was fixed by #2789
